packages/zipkin/src/InetAddress.js

```diff
@@ -21,6 +19,8 @@ class InetAddress {
 }
 
 InetAddress.getLocalAddress = function getLocalAddress() {
+  // eslint-disable-line global-require
+  const networkAddress = require('network-address');
```

Comment:
what happens when this is called in a browser? :) is there an alternative or fallback approach we can use in browsers?